base/linalg/uniformscaling.jl

```diff
@@ -138,6 +138,7 @@ function (-)(J::UniformScaling{TJ}, A::AbstractMatrix{TA}) where {TA,TJ<:Number}
 end
 
 inv(J::UniformScaling) = UniformScaling(inv(J.λ))
+norm(J::UniformScaling, p::Real=2) = abs(J.λ)
```

Comment:
AFAIK this check is there because arbitrary p-norms are hard to compute, but in this case the above is correct for all values of p (and mathematically makes sense for p >= 1). The doc does say p=1,2 or Inf though, but it's basically documenting an incompleteness in the implementation, so I'd think this is OK?